device-types/NEC/UNIVERGE-IX2105.yaml

```diff
@@ -0,0 +1,31 @@
+---
+manufacturer: NEC
+model: UNIVERGE IX2105
+slug: univerge-ix2105
+part_number: BE108821
+comments: |
+  NEC UNIVERGE IX2105, VPN-Compatible High speed Access Router, 7W
+  (1) RJ-45 Console Port, (1) Ethernet 1000BASE-T, (4) Ethernet 1000BASE-T in Switching Hub
+
+  Dimentions(WxDxH in mm): 135x196x36
+u_height: 1
+is_full_depth: false
+console-ports:
+  - name: CONSOLE
+    type: rj-45
+interfaces:
+  - name: GigaEthernet0
+    type: 1000base-t
+  - name: GigaEthernet1 port 1
```

Comment:
Are these actually referred to as `GigaEthernet1 port 1`? You should define these as they appear in the config.. If this is correct, please let me know

Reply:
`GigaEthernet1 port 1` is the actual interface name.
Here are the results of `show devices`.

```
ix2105# show devices
Device GigaEthernet0 is up
  ifIndex is 1
  GigaEthernet status:
    Physical address is 00:60:b9:xx:xx:xx
    Port status is up
    Full-duplex, 1G b/s, 1000BaseT, MDI
....
Device GigaEthernet1 is up
  ifIndex is 2
  GigaEthernet status:
    Physical address is 00:60:b9:xx:xx:xx
    Promiscuous mode is disabled
    Extended card is switching hub
      Port 1 status is up
        Full-duplex, 1G b/s, 1000BaseT, MDI-X
        VLAN-ID N/A, LAG-ID N/A
        0 CRC errors, 0 collisions
      Port 2 status is down
        Full-duplex, 1G b/s, 1000BaseT, MDI-X
        VLAN-ID N/A, LAG-ID N/A
        0 CRC errors, 0 collisions
      Port 3 status is down
        Full-duplex, 1G b/s, 1000BaseT, MDI-X
        VLAN-ID N/A, LAG-ID N/A
        0 CRC errors, 0 collisions
      Port 4 status is down
        Full-duplex, 1G b/s, 1000BaseT, MDI-X
        VLAN-ID N/A, LAG-ID N/A
        0 CRC errors, 0 collisions
    Entered multicast address 0 peak, 0 current
    Entered vrrp address 0 peak, 0 current
    Statistics:
....
```